Pulumi.website-dao-prd.yml

```diff
@@ -14,13 +14,11 @@ config:
   website-dao:GATSBY_BUY_MANA_URL: https://account.decentraland.org/
   website-dao:GATSBY_BUY_NAME_URL: https://builder.decentraland.org/names
   website-dao:GATSBY_DECENTRALAND_API: https://decentraland.org
+  website-dao:GATSBY_DCL_DATA_API: https://data.decentraland.vote/
   website-dao:GATSBY_DEFAULT_CHAIN_ID: "1"
   website-dao:GATSBY_DISCOURSE_API: https://forum.decentraland.org
   website-dao:GATSBY_DISCOURSE_CATEGORY: "13"
   website-dao:GATSBY_DISCOURSE_USER: dao
-  website-dao:GATSBY_DURATION_GRANT_TIER4: "1209600"
```

Comment:
This shouldn't be removed

Reply:
those are duplicated on master:

https://github.com/decentraland/governance/blob/master/Pulumi.website-dao-prd.yml#L44